### PR TITLE
QPPA-4395 - added content to `reference` section

### PR DESCRIPTION
**Pull request by peteplays**
[QPPA-4395](https://jira.cms.gov/browse/QPPA-4395)

- added content to `reference` section

### CODE REVIEW

File: src/components/references/references.js [hunk 1]
```diff
@@ -4,7 +4,60 @@ class References extends React.Component {
   render() {
     return (
       <div>
-        <h3>References</h3>
+        <h2 className='ds-h2' id='references'>References</h2>
+
+        <h3 className='ds-h3'>General References</h3>
+        <ul>
+          <li>
+            Resource Library: <a href='https://qpp.cms.gov/about/resource-library' target='_blank' rel='noopener noreferrer'>qpp.cms.gov/about/resource-library</a>
```
Comment (dschraitle) [inline src/components/references/references.js:12]: should these links be `ExternalLink` components? Or am I thinking of the dev-preview docs?

File: src/components/references/references.js [hunk 2]
```diff
@@ -4,7 +4,60 @@ class References extends React.Component {
...
+          </li>
+          <li>
+            Developer Group for QPP APIs: <a href='https://groups.google.com/forum/#!forum/qpp-apis' target='_blank' rel='noopener noreferrer'>groups.google.com/forum/#!forum/qpp-apis</a>
```
Comment (dschraitle) [inline src/components/references/references.js:15]: should there be a note here about this group not being active anymore?

File: src/components/references/references.js (hunk 1 above)
Comment (peteplays) [reply src/components/references/references.js:12]: Thats in the other repo.  when i upgrade this to react 16 ill create `ExternalLink`

File: src/components/references/references.js (hunk 2 above)
Comment (peteplays) [reply src/components/references/references.js:15]: ill ask Michelle

**Pull request merged**